unit-test

```diff
@@ -12,23 +12,36 @@ set -x
 
 build=$(mktemp -d)
 
+echo "## Compiling aklite"
 cmake -S . -B $build -DCMAKE_BUILD_TYPE=Debug -GNinja -DBUILD_P11=ON
 cmake --build $build --target aklite-tests
 
-# do additional clang-tidying in the aktualizr way
+echo "## Running clang tidy checks"
 clang_tests=$(ninja -C $build help | grep aktualizr_clang_tidy-src- | cut -d: -f1)
 cmake --build $build --target $clang_tests
 
 
-# do clang-formating in the aktualizr way
+echo "## Running clang format checks"
 clang_tests=$(ninja -C $build help | grep aktualizr_ci_clang_format-src- | cut -d: -f1)
 cmake --build $build --target $clang_tests
 
+echo "## Running aklite unit tests"
 cd $build
 ctest -j6 -V -L aklite
+ninja install
 cd $here
 rm -rf $build
 
+echo "## Compiling the custom-client-cxx example"
+build=$(mktemp -d)
+cp -r $here/aktualizr/third_party/jsoncpp/include/json /usr/include
```

Comment:
It means that this script (unit-test) can be executed only as a root and within a container.

Reply:
I thought that was already a requirement because the unit test runs wrapdocker - https://github.com/foundriesio/aktualizr-lite/blob/master/dev-shell#L24